tests/components/plex/helpers.py

```diff
@@ -0,0 +1,9 @@
+"""Helper methods for Plex tests."""
+from homeassistant.components.plex.const import DOMAIN, WEBSOCKETS
+
+
+def trigger_plex_update(hass, plex_server):
+    """Call the websocket callback method."""
+    server_id = plex_server.machineIdentifier
+    websocket = hass.data[DOMAIN][WEBSOCKETS][server_id]
```

Comment:
It would be better to patch the websocket and get access to the callback via the patch call and then call the callback to trigger an update. We don't want to use integration details like this in the tests.

Reply:
Done in #37147